test/fixtures/schema/array-fixed-inline-samples.json

```diff
@@ -185,7 +185,7 @@
                                   "content": "application/json"
                                 }
                               },
-                              "content": "{\n  \"address\": [\n    \"city\"\n  ]\n}"
+                              "content": "{\n  \"address\": [\n    \"street\",\n    \"city\",\n    \"state\"\n  ]\n}"
```

Comment:
Outside of the scope of this PR to fix. While reviewing this fixture. I believe the generated JSON Schema is incorrect.

Given what the `**` represent in fixed array is that the value is no longer required although the JSON Schema does require a min of 3 items. Shouldn't it be possible that the existing value of `["city"]` would validate given the MSON syntax as a valid value?

```json
{
  "$schema": "http://json-schema.org/draft-04/schema#",
  "type": "object",
  "properties": {
    "address": {
      "type": "array",
      "minItems": 3,
      "items": [
        {
          "type": "string"
        },
        {
          "type": "string",
          "enum": [
            "city"
          ]
        },
        {
          "type": "string"
        }
      ],
      "additionalItems": false
    }
  }
}
```

![screen shot 2018-07-17 at 18 54 12](https://user-images.githubusercontent.com/44164/42836394-d4163f88-89f2-11e8-9066-e1da2a3524e4.png)

Reply:
I actually don't see anything wrong here either in the body or the schema.